smoke/LeafDevice/details/Details.cs

```diff
@@ -104,6 +106,7 @@ async Task CreateDeviceIdentity(RegistryManager rm)
             this.context = new DeviceContext
             {
                 Device = device,
+                DeviceClientInstance = null,
```

Comment:
This doesn't seem like a good idea..

Reply:
Option<DeviceClient>

Reply:
Thanks! 

---
In reply to: [219926180](https://github.com/Azure/iotedge/pull/344#discussion_r219926180) [](ancestors = 219926180)